site/src/Consumer/UpdateProjectInformation.php

```diff
@@ -90,6 +117,21 @@ protected function processProject(Project $project) : bool
                 // Ignore -- we're already in the error handling path.
                 // The project will most likely remain in the "in processing"
                 // state.
+                // Send a notification that the project cannot be processed
+                $users = new ArrayCollection();
+                if ($project->getParentOrganization()) {
+                    $users = $project->getParentOrganization()->getMemberUsers();
+                } else {
+                    $users->add($project->getParentUser());
+                }
+                foreach ($users as $user) {
```

Comment:
How about adding a helper function to the NotificationCreatorService class, e.g. `notifyProjectOwners($project, $notification)`, that wraps this functionality?